src/plugins/auto/auto_schedule.cpp

```diff
@@ -113,40 +113,53 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
+        } else {
+            // When the hint is ctput and there are more than one device, the MULTI logic is used
+            std::string deviceName = "MULTI:";
+            for (auto& device : validDevices) {
+                deviceName += device.deviceName;
+                deviceName += ((device.deviceName == validDevices.back().deviceName) ? "" : ",");
+            }
+            _loadContext[ACTUALDEVICE].deviceInfo.deviceName = deviceName;
+            _loadContext[ACTUALDEVICE].deviceInfo.config[CONFIG_KEY(PERFORMANCE_HINT)] =
+                InferenceEngine::PluginConfigParams::CUMULATIVE_THROUGHPUT;
+            _loadContext[ACTUALDEVICE].deviceInfo.config[CONFIG_KEY(PERF_COUNT)] =
+                _autoSContext->_needPerfCounters ? InferenceEngine::PluginConfigParams::YES
+                                                 : InferenceEngine::PluginConfigParams::NO;
+            if (_autoSContext->_bindBuffer)
+                _loadContext[ACTUALDEVICE].deviceInfo.config[ov::intel_auto::device_bind_buffer.name()] =
+                    InferenceEngine::PluginConfigParams::YES;
         }
-        _loadContext[ACTUALDEVICE].deviceInfo.deviceName = deviceName;
-        _loadContext[ACTUALDEVICE].deviceInfo.config[CONFIG_KEY(PERFORMANCE_HINT)] =
-            InferenceEngine::PluginConfigParams::CUMULATIVE_THROUGHPUT;
-        _loadContext[ACTUALDEVICE].deviceInfo.config[CONFIG_KEY(PERF_COUNT)] =
-            _autoSContext->_needPerfCounters ? InferenceEngine::PluginConfigParams::YES
-                                             : InferenceEngine::PluginConfigParams::NO;
-        if (_autoSContext->_bindBuffer)
-            _loadContext[ACTUALDEVICE].deviceInfo.config[ov::intel_auto::device_bind_buffer.name()] = InferenceEngine::PluginConfigParams::YES;
     } else {
-        _loadContext[ACTUALDEVICE].deviceInfo = _autoSContext->_plugin->SelectDevice(_autoSContext->_devicePriorities,
-                                                                           _loadContext[ACTUALDEVICE].networkPrecision,
-                                                                           _autoSContext->_modelPriority);
+        _loadContext[ACTUALDEVICE].deviceInfo =
+            _autoSContext->_plugin->SelectDevice(_autoSContext->_devicePriorities,
+                                                 _loadContext[ACTUALDEVICE].networkPrecision,
+                                                 _autoSContext->_modelPriority);
     }
     LOG_INFO_TAG("select device:%s", _loadContext[ACTUALDEVICE].deviceInfo.deviceName.c_str());
     bool isActualDevCPU =
         _loadContext[ACTUALDEVICE].deviceInfo.deviceName.find("CPU") !=std::string::npos && !isCumulative;
-    // if Actual device is CPU, disabled _loadContext[CPU], only use _loadContext[ACTUALDEVICE]
+    // if Actual device is CPU or perf_hint is cumulative, disabled _loadContext[CPU], only use _loadContext[ACTUALDEVICE]
     if (isActualDevCPU || isCumulative) {
```

Comment:
AUTO:GPU with cumulative, is cpu acceleration expected?

Reply:
AUTO:GPU with cumulative, it won't enable cpu acceleration. I will add some test cases to cover this logic.

Reply:
Updated test case